slack-client/index.js

```diff
@@ -28,11 +23,14 @@ app.event('channel_id_changed', ChannelHandler.channel_id_changed);
 app.event('message', MessageHandler.channel_message);
 
 async function startApp() {
-	await prepopulateDBs(WebClient)
+	await prepopulateDBs(SlackWebClient)
 		.then(arr_of_responses => console.log(arr_of_responses))
```

Comment:
arr_of_responses -> responses or responses_list
usually var names dont need the data type